spec/pods.md

```diff
@@ -179,3 +180,7 @@ JSON Schema for the Pod Manifest, conforming to [RFC4627](https://tools.ietf.org
 * **ports** (list of objects, optional) list of ports that SHOULD be exposed on the host.
     * **name** (string, required, restricted to the [AC Name](#ac-name-type) formatting) name of the port to be exposed on the host. This field is a key referencing by name ports specified in the Image Manifest(s) of the app(s) within this Pod Manifest; consequently, port names MUST be unique among apps within a pod.
     * **hostPort** (integer, required) port number on the host that will be mapped to the application port.
+* **exitPolicy** (string, optional) a string that specify the exit policy of the pod, if left empty, then it's up to ACE to choose the default behavior. Valid values are:
```

Comment:
Please note that I am not saying you should change it to be like
kubernetes.  Consider it in fresh light.  I thing RestartPolicy is clearer
than ExitPolicy, but I think RunPolicy *might* be even better.

On Fri, Sep 25, 2015 at 4:40 PM, Yifan Gu <notifications@github.com> wrote:

> In spec/pods.md
> <https://github.com/appc/spec/pull/500#discussion_r40485441>:
>
> > @@ -179,3 +180,7 @@ JSON Schema for the Pod Manifest, conforming to [RFC4627](https://tools.ietf.org
> >  * **ports** (list of objects, optional) list of ports that SHOULD be exposed on the host.
> >      * **name** (string, required, restricted to the [AC Name](#ac-name-type) formatting) name of the port to be exposed on the host. This field is a key referencing by name ports specified in the Image Manifest(s) of the app(s) within this Pod Manifest; consequently, port names MUST be unique among apps within a pod.
> >      * **hostPort** (integer, required) port number on the host that will be mapped to the application port.
> > +* **exitPolicy** (string, optional) a string that specify the exit policy of the pod, if left empty, then it's up to ACE to choose the default behavior. Valid values are:
>
> Sure, I am happy with changing this to restart policy. Waiting for other
> maintainers' feedback.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appc/spec/pull/500/files#r40485441>.
>